content/en/blog/the_sandbox_launch.md

```diff
@@ -0,0 +1,54 @@
+---
+title : "The ODC Sandbox Launch"
+description: "OpenData Community Hackathon: Partners host educational Twitter Spaces"
+lead: "Fight Sybils, Decentralize Data"
+contributors: ["Stefi"]
+date: 2023-02-19T22:26:54+01:00
+lastmod: 2023-02-19T22:26:54+01:00
+draft: false
+images: []
+---
+
+Our last hackathon occurred from Jan 5th till Jan 31st , competitors collaborated to 4 different bounties, each with a different focus like helping the community growth, developing dashboards,
+identifying sybil behaviors through exploratory analysis and packaging algorithms in “Legos” that can be chained together with the intention to collaborate 
+for an accurate classification of sybil wallets.  Details about the competition can be found in this [post](https://opendatacommunity.org/blog/data-builder-hackathon/) about the hackathon opening.  
+
+We had an intense month with projects feedback sections together with FDD analysts, twitter spaces with sponsors and a lot more
+. What about the results? Incredible! We had 53 submissions and 29 winning projects that shared 40K in prizes, the projects range from python packages to 
+whole decentralized data frameworks that can integrate live dashboards, details and other results can be here on this [hackathon wrap up presentation](https://docs.google.com/presentation/d/1Ic4p9o2pLHETjBv1N9v4wkLTYToZxh-GP6Bo8ICtkko/edit#slide=id.g20c507cff6c_0_1)
+Most important of all, some projects developed in the hackathon demonstrate great value, scalability and potential to be strong weapons in the fight against sybil 
+attacks, mainly in the distribution of public funds.  
+
+Therefore, we are very pleased to announce the birth of the OpenData Community Sandbox, a space to foster the maturation of these projects.
+In the coming weeks, the community will discuss which will be the initial projects invited to join the Sandbox, 
+these projects will be able to rely on numerous benefits such as:
+
+**Tooling**
+- Repositories inside ODC github   
+- Dedicated Discord channel  
+- Educational help on setting tools up   
+  
+**Promotion**   
+- web site listings   
+- Twitter spaces   
+- Ongoing promotion through ODC channels   
+- Featured inclusion in the [ODC Landscape](https://opendatacommunity.org/docs/landscape/)
+- Featured inclusion in our data stack   
+
+**Technical help**
+- Mentorship   
+- Feature in Hackathons   
+- Ongoing hackathon - aka bounties to accelerate and attract contributions  
+
+**Network**  
+- Community builders engagement and contributions (including some FDD/Gitcoin members) 
```

Comment:
replace with -community builders and contributors passionate about an open data future